tests/integration/controller/kyma/kyma_test.go

```diff
@@ -176,6 +177,33 @@ var _ = Describe("Kyma enable one Module", Ordered, func() {
 	})
 })
 
+var _ = Describe("Kyma enable one Mandatory Module", Ordered, func() {
+	kyma := NewTestKyma("mandatory-module-kyma")
+
+	RegisterDefaultLifecycleForKyma(kyma)
+
+	It("should result Kyma in Error state", func() {
+		By("enabling one mandatory Module")
+		kyma.Spec.Modules = append(kyma.Spec.Modules, v1beta2.Module{
+			Name:    "mandatory-template-operator",
+			Channel: "mandatory",
```

Comment:
although this channel should be irrelevant, we should design a test based on the real use case, we don't have a channel `mandatory`, if assign a module to `spec.modules`, either use a default regular channel or simply not define the channel but relies on kyma global channel.